srcpkgs/rundird/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'rundird'
+pkgname=rundird
+version=0.1.1
+revision=1
+build_style=zig-build
```

Comment:
Why `zig-build` and not just `zig`? Because the command used is `zig build`?

Reply:
From the new commit message:

We call this "zig-build" instead of just "zig" as this build-style
relies on usage of the zig build system. In the future, other build
systems such as meson may support zig code as well. Furthermore, the
zig build system may be used to build C/C++ code as well, not just zig.

Reply:
That makes sense, thanks :)